core/src/App.html

```diff
@@ -246,7 +246,17 @@
       console.info('iframe does not exist, not able to send context.');
       return;
     }
-    const internal = prepareInternalData(config);
+
+    let internal = prepareInternalData(config);
+    const userSettingsGroupName =
+      config.iframe.luigi.currentNode.userSettingsGroup;
+
+    if (userSettingsGroupName) {
+      const userSettingGroups = await LuigiConfig.readUserSettings();
+      const userSettings = userSettingGroups[userSettingsGroupName];
```

Comment:
[To the previous thread](https://github.com/SAP/luigi/pull/1754/files#r544245121): `return localStorageValue && JSON.parse(localStorageValue);`

That would indeed return false/null instead of possible undefined, but you'd still be trying to access `userSettingsGroupName`  of a null value. 
For me it still shows **TypeError: Cannot read property 'userAccount' of null**. I removed the cached when I retested to check the behavior, probably you have it still saved

Reply:
Thanks, @ndricimrr for pointing this out. I push a fix.